<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookmarkDialog</class>
 <widget class="QDialog" name="bookmarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bookmark It</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/app/icons/app-icons/Bookmarks.svg</normaloff>:/app/icons/app-icons/Bookmarks.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QLabel" name="path">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="bkName"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="Lpath">
     <property name="text">
      <string>Path :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="LbkSection">
     <property name="text">
      <string>Section</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="bkSection"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="LbkName">
     <property name="text">
      <string>Bookmark Name :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="bkStatus">
     <property name="styleSheet">
      <string notr="true">color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string>bkStatus</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="done">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="QLabel" name="bkIcon">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>120</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
